packages/public/server/src/module/Authentication/src/Service/AuthenticationService.php

```diff
@@ -128,6 +142,11 @@ public function hasIdentity()
         return $return;
     }
 
+    public function setIdentity(UserInterface $identity)
+    {
+        $this->identity = $identity;
+    }
```

Comment:
This is used to simulate that `$identity` is authenticated in the current session.